jetbrains-core/resources/icons/misc/csharp.svg

```diff
@@ -0,0 +1,10 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
```

Comment:
do we have proper attribution on these SVGs

Reply:
all the icons are downloaded from https://jetbrains.design/intellij/resources/icons_list, with content unchanged. so if it doesn't have an atttribution here it doesn't have one.